Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.4-210 as builder
```

Comment:
do we want to use this or quay?

Reply:
Since this is ultimately a downstream dockerfile, I could see it either way.